### PR TITLE
Added update profile picture feature with popover

### PR DESCRIPTION
**Pull request by anurag0singh**
# Related Issue
Fixes: #120 

# Prosposed Changes
- change 1
Screenshot (134)

- change 2
Screenshot (135)

# Additional Info
- These popover features will be available in students as well as volunteers update pages.

### CODE REVIEW

**Review by garg3133 — COMMENTED** [review]
Few more points:
* The options should be hoverable (pointer cursor and a darker background on hover)
* The file field should be hidden as we are not directly using it.
* The idea is not to select the image in an existing file field and then click on Submit button but to change the profile pic (and only profile pic) there and then just as we select an image. For this, you can create a separate form for updating the image and as soon as an image is selected (making sure it is an image first, refer to #130), submit that form. See how can you distinguish in the view from which form the request is coming.

File: templates/sidebar.html [hunk 1]
```diff
@@ -116,17 +116,31 @@
...
-                            {% if profile.profile_image %}
-                            <img class="sidebar-img rounded-circle my-2" src='{{ profile.profile_image.url }}' alt="v">
-                            {% elif profile.gender == 'M' %}
-                            <img class="sidebar-img rounded-circle my-2" src="{% static 'home/images/man.png' %}" alt="v">
-                            {% elif profile.gender == 'F' %}
-                            <img class="sidebar-img rounded-circle my-2" src="{% static 'home/images/woman.png' %}" alt="v">
-                            {% else %}
-                            <img class="sidebar-img my-2" src="{% static 'home/logo.png' %}" alt="v">
-                            {% endif %}
+                            <button type="button" class="btn rounded-circle example-popover p-1" data-container="body" data-toggle="popover" data-placement="bottom"
```
[the remark this answers is not in the document]
Comment (anurag0singh) [reply templates/sidebar.html:119]: @garg3133  Should I hide the upload file in the volunteers update profile too or just changing it in the student's profile update will suffice?

[the remark this answers is not in the document]
Comment (garg3133) [reply templates/sidebar.html:119]: Do one thing, don't add this feature on the update volunteer profile. We're going to change the update profile page anyway sometime soon to make it look like update student profile page.

**Comment by anurag0singh** [comment]
@garg3133 I have created another PR #153 with all the changes you suggested. Please check it.

**Pull request closed without merging**